src/pages/payment-shipping/payment-shipping.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import parse from 'html-react-parser';
+
+import { useStyles } from './payment-shipping.style';
+import { getBusinessPageByCode } from '../../redux/businessPages/businessPages.actions';
+
+const PaymentsAndShipping = () => {
+  const dispatch = useDispatch();
+  const { paymentAndShippingPage, language } = useSelector(
+    ({ BusinessPages, Language }) => ({
+      paymentAndShippingPage: BusinessPages.pages.paymentAndShipping,
+      language: Language.language
+    })
+  );
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+    dispatch(getBusinessPageByCode('payment-and-shipping'));
+  }, [dispatch]);
+
+  const PaymentsAndShippingText = Object.keys(paymentAndShippingPage).length
```

Comment:
Do we really need this check? I think business pages should always be translated, it should be ensured on the backend and admin side. Here we should not care about it

Reply:
Please, lowercase the name of the variable, it is not a component, it is just a text